v3/src/components/case-tile-common/attribute-header.tsx

```diff
@@ -60,6 +62,7 @@ export const AttributeHeader = observer(function AttributeHeader({
     prefix: instanceId, dataSet: data, attributeId
   }
   const { attributes, listeners, setNodeRef: setDragNodeRef } = useDraggableAttribute(draggableOptions)
+  useOutsidePointerDown({ ref: menuListRef, handler: () => onCloseMenuRef.current?.() })
```

Comment:
Looks fine (with `menuListRef` rather than `inputRef`, but I note that according to the Chakra docs there's an `onPointerDownOutside` prop to `Menu` that would be worth trying rather than replicating a built-in capability.

Reply:
The `closeOnBlur` prop closes the menu on pointerup. So menu stays open while user is resizing.

Reply:
Apparently I was looking at the v3 Chakra docs, which adds the `onPointerDownOutside` prop to `Menu`, so I guess we're stuck with our own implementation for now.